src/scripts/underground/Underground.ts

```diff
@@ -88,6 +112,22 @@ class Underground implements Feature {
         return Underground.BASE_BOMB_EFFICIENCY + this.getUpgrade(UndergroundUpgrade.Upgrades.Bomb_Efficiency).calculateBonus();
     }
 
+    getSurvey_Cost() {
+        return Underground.SURVEY_ENERGY - this.getUpgrade(UndergroundUpgrade.Upgrades.Survey_Cost).calculateBonus();
+    }
+
+    getSurvey_Efficiency() {
+        return Underground.BASE_SURVEY_CHARGE_EFFICIENCY + this.getUpgrade(UndergroundUpgrade.Upgrades.Survey_Efficiency).calculateBonus();
+    }
+
+    getNewYLayer() {
```

Comment:
Can we change this to be a little more descriptive of it's actual function,
Possibly one of these:
```suggestion
    getYLayers() {
```
```suggestion
    getTotalYLayers() {
```
```suggestion
    getMineHeight() {
```